gitlab-cng-17.1.yaml

```diff
@@ -233,7 +237,7 @@ subpackages:
             # Stop the registry
             kill $PID
 
-  - name: ${{package.name}}-gitlab-container-registry-compat
+  - name: ${{package.name}}-container-registry-compat
```

Comment:
Should we use the var-transforms approach here and append only the version number at the end? 

i.e this change would produce:
gitlab-cng-17.1-container-registry-compat

whereas we ideally want
gitlab-cng-container-registry-compat-17.1

Reply:
I agree, that's a lot better. I'll get a commit up

Reply:
Realized in the process that if we're going to track the version that way, we can eliminate `gitlab-cng` from the package name entirely too